specs/Checkbox.md

```diff
@@ -0,0 +1,388 @@
+
+### Differences of Fabric/Stardust to resolve
+
+| Name                 | Type                                                        | Notes                                                                          |
+| -------------------- | --------------------------------------------------------    | -------------------------------------------------------------------------------|
+| animation            | AnimationProp                                               |                                                                                |
+| disableAnimations    | boolean                                                     | default false                                                                  |
+| overwrite            | boolean                                                     | default false                                                                  |
+| renderer             | Renderer                                                    |                                                                                |
+| variables            | any                                                         |                                                                                |
+| target               | Document                                                    |                                                                                |
+| content              | ReactNode                                                   |                                                                                |
+| ariaPositionInSet    | number                                                      | if checkbox is in a set, should be up to the user to provide a11y              |
+| ariaSetSize          | number                                                      |                                                                                |
+
+### Conversion process from Fabric 7 to Fluent UI Checkbox
+
+Props being changed:
+
+Some props, like style & className, should always go into the root or to the applicable element like name into the box element (replacing input).
+Data-attributes will be spread using slotProps. 
+
+Props being removed:
+
+ariaPoisitionInSet and ariaSetSize - when writing parent component, user should set these on the checkbox.
+animations: remnant pattern of semantic UI, don't need animations for checkbox theming
+defaultChecked: overloading with checked - can just set default value of checked.
+
+## Slots
+
+| Name      | Considerations                                         |
+| --------- | ------------------------------------------------------ |
+| root      | label                                                  |
+| input     | actual checkbox element - what gets checked/unchecked  |
+| icon      | visual checkmark, sideways if indeterminate            |
+| box       | wraps input & icon - what actual gets the styling      |
```

Comment:
GROUP REVIEW:

What are our design principles for slots?  Inline replaceable? Can I add a slot to a component?  Etc.